src/test/java/org/example/c20debugging/BankAccountTest.java

```diff
@@ -0,0 +1,64 @@
+package org.example.c20debugging;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class BankAccountTest {
+    private BankAccount account;
+    private static int count;
+
+
+    public static void beforeClass() {
+        System.out.println("This executes before any test cases. Count = " + count++);
+    }
+
+    @org.junit.Before
```

Comment:
Use `org.junit.jupiter` imports instead of `org.junit`

Reply:
done